### ISSUE TITLE
images

### ISSUE THREAD

**Issue by vlast3k** [vlast3k/vESPrino#3]

**Comment by vlast3k** [comment]
![helen](https://user-images.githubusercontent.com/492455/111103866-10c69c80-8558-11eb-84f2-56631fc337b1.PNG)
![hufnagl](https://user-images.githubusercontent.com/492455/111103885-191ed780-8558-11eb-95a8-57b4b89816b7.jpg)

**Comment by vlast3k** [comment]
![hufnagl](https://user-images.githubusercontent.com/492455/111103943-36ec3c80-8558-11eb-8025-af20c7f6fe91.jpg)

**Comment by vlast3k** [comment]
![Radoslav-Nikolov-top](https://user-images.githubusercontent.com/492455/111104717-f8578180-8559-11eb-932a-f96d7cff9955.jpg)

**Comment by vlast3k** [comment]
![twitter-photo](https://user-images.githubusercontent.com/492455/111482304-72476080-873c-11eb-83fa-cc17b0dc1bc4.jpg)

**Comment by dkom2021sofia** [comment]
![Michael Weingartner Headshot](https://user-images.githubusercontent.com/80735012/112151403-dc06b500-8be9-11eb-9684-dd15ee24035a.jpg)

**Comment by dkom2021sofia** [comment]
![Michael Weingartner Headshot](https://user-images.githubusercontent.com/80735012/112151772-44559680-8bea-11eb-9429-0edb9e00f119.jpg)

**Comment by dkom2021sofia** [comment]
<img width="126" alt="Screenshot 2021-03-24 at 18 44 38" src="https://user-images.githubusercontent.com/80735012/112403442-17eb6880-8d17-11eb-90d5-c07b0429aa42.png">

**Comment by ghost** [comment]
wtf...

**Comment by ghost** [comment]
Bitcoin...? <https://www.youtube.com/watch?v=5ywjpbThDpE>

**Comment by ghost** [comment]
Cat yodeling...? <https://www.youtube.com/watch?v=cxLG2wtE7TM>

**Comment by ghost** [comment]
Aaahh....

Purrminator.... <https://www.youtube.com/watch?v=1QdfT0nw6XM>

**Comment by ghost** [comment]
No... the olympics <https://www.youtube.com/watch?v=tbXsP2KlR2A>...

pfff (yodel)

What's up vlad?
